luigi_pipeline/lib/hail_tasks.py

```diff
@@ -18,22 +18,86 @@
 
 logger = logging.getLogger(__name__)
 
+GRCh37_STANDARD_CONTIGS = {
```

Comment:
I think it'll be easier to review this if we split the `black` formatting and moving `contig_check` & `validate_mt` into its own PR?

Reply:
Yeah that is fair, I'll split this and label it as stacked -- my VSCode formats with black on save so that explains the ".